packages/hardhat-core/src/internal/core/providers/accounts.ts

```diff
@@ -253,6 +253,7 @@ export class LocalAccountsProvider extends ProviderWrapperWithChainId {
       gasLimit: transactionRequest.gas,
     };
 
+    // TODO: consider changing instances of "london" below to ["latest hardfork"]
```

Comment:
London here is just a way to signal to ethereumjs that EIP-1559 may be used

Reply:
On a call, Pato said this is fine as-is, TODO and all.